Samples/login.php

```diff
@@ -38,7 +38,7 @@
  * @param \Fhp\BaseAction $action Some action that requires strong authentication.
  * @throws CurlException|UnexpectedResponseException|ServerException See {@link FinTs::execute()} for details.
  */
-function handleStrongAuthentication(\Fhp\BaseAction $action)
```

Comment:
Did you remove that leading back slash on purpose?

Reply:
added it again